commands/utils/docker-endpoint.ts

```diff
@@ -9,14 +9,8 @@ class DockerClient {
     private endPoint:Docker;
 
     constructor() {
-        if (process.env.DOCKER_HOST){
-            this.endPoint = new Docker();
-        }
-        else if (process.platform === 'win32') {
-            this.endPoint = new Docker({ socketPath: "//./pipe/docker_engine" });
-        } else {
-            this.endPoint = new Docker({ socketPath: '/var/run/docker.sock' });
-        }
+        // Pass no options so that the defaultOpts of docker-modem will be used
+        this.endPoint = new Docker(null);
```

Comment:
we crafted the typings file  by hand, so go ahead and make a local change to it and include it in the PR